tests/0.2.0.json

```diff
@@ -0,0 +1,377 @@
+{
```

Comment:
After this JSON Schema is reviewed and approved in this PR, I will remove this from this PR and add it to `recap-website/specs/type/0.2.0.json`, then switch from a file reference to a network call.

I'm including it in this PR draft to make it easier to iterate on the spec and tests in parallel.